pcdsdevices/ipm.py

```diff
@@ -174,6 +194,9 @@ class IPIMB(Device):
     delay: delay of trigger relative to input EVR
     trigger:
     """
+
+    tab_whitelist = ['isum', 'xpos', 'ypos']
+
     isum = Cpt(EpicsSignal, ':SUM', kind='hinted')
```

Comment:
should all these be digitizer signals `EpicsSignalRO`...?

Reply:
Yeah, they totally should :/

Reply:
Talked with Mike and sorted it out. Will push new read-onlys plus an extra configuration PV for Wave8 soon.